Removing conda
Loading conda
Wed Feb  5 03:10:42 UTC 2025
(353088,)
type: normal
NHDPLUS_H_0108_HU4_GDB
flowlines read-in
exploded
type: normal
NHDPLUS_H_0110_HU4_GDB
flowlines read-in
exploded
type: normal
NHDPLUS_H_0202_HU4_GDB
flowlines read-in
exploded
INFO: Pandarallel will run on 128 workers.
INFO: Pandarallel will use Memory file system to transfer data between the main process and workers.
/work/pi_cjgleason_umass_edu/.conda/envs/narrowPIXC/lib/python3.12/site-packages/geopandas/geodataframe.py:1819: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  super().__setitem__(key, value)
Indices: [4, 6, 7]
type: segmented
NHDPLUS_H_0108_HU4_GDB
/nas/cee-water/cjgleason/fiona/narrow_rivers_PIXC_data/NHD_prepped_segmented/HUC2_01/NHDPLUS_H_0108_HU4_GDB_prepped_segmented.parquet
segments read-in
type: segmented
NHDPLUS_H_0110_HU4_GDB
/nas/cee-water/cjgleason/fiona/narrow_rivers_PIXC_data/NHD_prepped_segmented/HUC2_01/NHDPLUS_H_0110_HU4_GDB_prepped_segmented.parquet
segments read-in
type: segmented
NHDPLUS_H_0202_HU4_GDB
/nas/cee-water/cjgleason/fiona/narrow_rivers_PIXC_data/NHD_prepped_segmented/HUC2_02/NHDPLUS_H_0202_HU4_GDB_prepped_segmented.parquet
segments read-in
Process ForkPoolWorker-524:
Traceback (most recent call last):
  File "/work/pi_cjgleason_umass_edu/.conda/envs/narrowPIXC/lib/python3.12/multiprocessing/pool.py", line 131, in worker
    put((job, i, result))
  File "/work/pi_cjgleason_umass_edu/.conda/envs/narrowPIXC/lib/python3.12/multiprocessing/queues.py", line 399, in put
    self._writer.send_bytes(obj)
  File "/work/pi_cjgleason_umass_edu/.conda/envs/narrowPIXC/lib/python3.12/multiprocessing/connection.py", line 200, in send_bytes
    self._send_bytes(m[offset:offset + size])
  File "/work/pi_cjgleason_umass_edu/.conda/envs/narrowPIXC/lib/python3.12/multiprocessing/connection.py", line 427, in _send_bytes
    self._send(header + buf)
  File "/work/pi_cjgleason_umass_edu/.conda/envs/narrowPIXC/lib/python3.12/multiprocessing/connection.py", line 384, in _send
    n = write(self._handle, buf)
        ^^^^^^^^^^^^^^^^^^^^^^^^
BrokenPipeError: [Errno 32] Broken pipe
/var/spool/slurm/slurmd/job28460478/slurm_script: line 21: 2376297 Killed                  python3 evalCoverage.py max
Wed Feb  5 03:13:43 UTC 2025
Script completed.
slurmstepd-ceewater-cpu009: error: Detected 3 oom_kill events in StepId=28460478.batch. Some of the step tasks have been OOM Killed.
